### PR TITLE
Add a basic ui for bookings

### PR DESCRIPTION
**Pull request by hanhuiice**

### CODE REVIEW

File: src/main/java/seedu/trackbeau/ui/BookingListPanel.java [hunk 1]
```diff
@@ -1,17 +1,47 @@
...
+     * Creates a {@code BookingListPanel} with the given {@code ObservableList}.
+     */
+    public BookingListPanel(ObservableList<Booking> bookingList) {
         super(FXML);
+        bookingListView.setItems(bookingList);
+        bookingListView.setCellFactory(listView -> new BookingListViewCell());
+    }
+
+    /**
+     * Custom {@code ListCell} that displays the graphics of a {@code Customer} using a {@code CustomerCard}.
```
Comment (flairekq) [inline src/main/java/seedu/trackbeau/ui/BookingListPanel.java:32]: Should be {@code Booking} using a {@code BookingCard}?

File: src/main/resources/view/BookingListCard.fxml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.RowConstraints?>
+<?import javafx.scene.layout.VBox?>
+
+<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (flairekq) [inline src/main/resources/view/BookingListCard.fxml:14]: i think need to change id="cardPane" to styleClass="cardPane" cause there's more than 1 element that is using id="cardPane".
`<HBox styleClass="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
`

**Review by flairekq — CHANGES_REQUESTED** [review]
Overall looks good just some cosmetic issues to fix

File: src/main/java/seedu/trackbeau/ui/MainWindow.java [hunk 3]
```diff
@@ -110,7 +112,8 @@ public void handle(MouseEvent event) {
                     detailsPanelPlaceholder.getChildren().add(customerListPanel.getRoot());
                     break;
                 case "bookingsLabel":
-                    bookingListPanel = new BookingListPanel();
+                    //bookingListPanel = new BookingListPanel(logic.getFilteredBookingList());
```
Comment (isabelteo) [inline src/main/java/seedu/trackbeau/ui/MainWindow.java:115]: dead code

**Review by isabelteo — COMMENTED** [review]
LGTM

**Review by flairekq — COMMENTED** [review]
LGTM!

**Review by flairekq — APPROVED** [review]

**Pull request merged**